linode/manager
# feat: [M3-6924] - Allow users to filter Databases by `Region` and `Engine`

## Description 📝

- Allows users to order their Databases by more columns 🎉⬆️⬇️
- The API just added this functionality so we should give users this ability to order by more columns!

## Major Changes 🔄
- Users can now order their Databases by `region` and `engine` in addition to the existing ordering options 🎉

## Preview 📷

### Before 😖

https://github.com/linode/manager/assets/115251059/ed1c5640-8f9e-4a07-a84b-2021b0868b0d

### After ✅

https://github.com/linode/manager/assets/115251059/63b9f5e5-5de1-4135-94e1-6ef1bea06815

## How to test 🧪

> **Warning**
> You **must** use the dev API to test this right now

1. Point your local Cloud Manager to the **dev API** (ask me if you don't know how)
2. **Create two Databases Clusters** (one PostgreSQL and one MySQL). You must choose _Newark, NJ USA_ as the region because of limitations in the dev environment  
4. Verify that you can **order your Databases by `Engine`** and other fields. Unfortunately we cannot test ordering by `Region` because in dev, only databases can exist in  _Newark, NJ USA_
